onnxruntime/test/perftest/ort_test_session.cc

```diff
@@ -546,6 +546,50 @@ select from 'TF8', 'TF16', 'UINT8', 'FLOAT', 'ITENSOR'. \n)");
   }
 }
 
+// seed=-1 means no initilize
+void OnnxRuntimeTestSession::initilize_tensor_with_seed(int32_t seed, ONNXTensorElementDataType type,
+                                                        void* data_ptr, const std::vector<int64_t>& input_node_dim) {
+  if (seed < 0) return;
+  std::default_random_engine engin;
```

Comment:
Can RandomValudGenerator in onnxruntime\test\common\tensor_op_test_utils.h be used?

Reply:
It could be. But we have to introduce a new lib "gtest" into perftest as 'gtest.h' is included in 'tensor_op_test_utils.h' . Does that make sence?